Formula/git-secret.rb

```diff
@@ -15,12 +17,44 @@ class GitSecret < Formula
   depends_on :gpg => :recommended
 
   def install
+    # Prefer GPG2 by default.
+    inreplace "src/_utils/_git_secret_tools.sh", ':="gpg"', ':="gpg2"'
```

Comment:
Perhaps worth noting if we rename `gpg` to `gpg1` and let `gpg2` symlink itself onto `gpg` we're going to end up with `inreplace`s anyway unless we drop support for `gpg` 1.x in the requirement completely. No upstream script/usage/etc is looking for a `gpg1` binary, as far as I saw during the rewrites and testing here.

The initial round of feedback makes me suspect simply saying "We won't support GPG 1.x any more" is perhaps a rational solution.